docs/qa/setup_local_qa_cops_test.rst

```diff
@@ -0,0 +1,177 @@
+.. _setup-local-qa-cops-testing:
+
+========================================================
+Setting up local qa cops test environs (will be updated)
+========================================================
+
+Objective
+---------
+
+Get familiar with Concourse by setting up the bakery-pipeline locally.
+This is the pipeline which is used under the hood for `Content Output Producer (COPS) <https://cops.openstax.org/>`_
+
+Retrieve Project & Config File
+------------------------------
+
+1. Clone repository containing the pipeline configuration file to your local directory:
+
+.. code-block:: bash
+
+   $ git clone git@github.com:openstax/concourse-pipelines.git
+
+2. Change to bakery directory to retrieve the correct pipeline (there are several other pipelines in this repository):
+
+.. code-block:: bash
+
+   $ cd bakery
+
+3. Check to see if the pipeline.yml exists in the this directory:
+
+.. code-block:: bash
+
+   $ ls
+
+You should see pipeline.yml listed. This file is used to configure the pipeline.
+
+Setup local concourse server
+----------------------------
+
+0. Prerequisites:
+
+- `Docker <https://www.docker.com/>`_
+
+- `fly CLI - Concourse command line tool <https://concourse-ci.org/fly.html>`_
+
+- `wget <https://www.gnu.org/software/wget/>`_
+
+Steps to install wget (as per INSTALL document in the directory):
+
+- download wget-1.20.3.tar.gz (or latest)
+
+- unzip and cd into wget directory
+
+- run brew install gnutls
+
+- run ./configure
+
+- run make
+
+- run make install
+
+- run make clean
+
+1. Retrieve the docker-compose file to set up concourse server locally.
+In the /bakery directory that contains your pipeline.yml, run:
+
+.. code-block:: bash
+
+   $ wget https://concourse-ci.org/docker-compose.yml
+
+2. Start your Concourse server locally:
+
+.. code-block:: bash
+
+   $ docker-compose up -d
+
+   Terminal output should show the following:
+   Creating network "bakery_default" with the default driver
+   Creating bakery_concourse-db_1 ... done
+   Creating bakery_concourse_1  ... done
+
+3. To see the pipelines in Concourse UI at work, navigate to:
+
+   `localhost:8080 <localhost:8080>`_
+
+Setup pipeline with local Concourse and pipeline.yml
+----------------------------------------------------
+
+1. Target the Concourse UI, so you are able to set the pipelines to UI:
+
+.. code-block:: bash
+
+   $ fly -t local-pipeline-stuff login -c http://localhost:8080 -u test -p test
```

Comment:
-t is for target which would be localhost. `local-pipeline-stuff` might be better renamed to `local`, `dev-local`, or something other than "local-pipeline-stuff".

Reply:
this is an example name by Brenda. I changed it to local and noted that it can be named by the dev/qa